.github/workflows/pr-labeler.yml

```diff
@@ -1,14 +0,0 @@
-name: Automatic Pull Request Labeler
-
-on:
-    pull_request_target:
-
-jobs:
-    pr_labeler:
-        name: Automatic Pull Request Labeler
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/labeler@main
-              with:
-                  repo-token: '${{ secrets.GITHUB_TOKEN }}'
-                  sync-labels: true
```

Comment:
Don't delete, rename to .disabled or something

Reply:
Yeah kinda too late, it's been removed in all the other repos already as well (Framework and pieces were approved by @kyranet)

Reply:
ree, fineee